frontend/src/Components/Poll/Poll.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { useCallback, useEffect, useState } from 'react'
+
+import DateComponent from '@components/DateComponent'
+import Button from '@ui/Button'
+import Checkbox from '@ui/Checkbox'
+import Radio from '@ui/Radio'
+import { toast } from 'react-toastify'
+
+import { useAPI } from '../../AppState/AppState'
+import PollService from '../../Services/PollService'
+import { Poll as PollType } from '../../Types/Poll'
+import { VotersTooltip } from './VotersList'
+
+import styles from './Poll.module.css'
+
+interface PollProps {
+  pollId: string
+}
+
+export const Poll: React.FC<PollProps> = ({ pollId }) => {
```

Comment:
Let's rename it to `PollComponent` or at least to `PollComp`

my rationale for that: when looking for particular UI component I usually search by {Name}Comp, that helps distinguish the result from the rest of Poll* entities 

what do you think?